src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,182 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public MyLinkedList() {
+        head = null;
+        tail = null;
+        size = 0;
+    }
+
     @Override
     public void add(T value) {
+        Node<T> lastNode = tail;
```

Comment:
And I have some question here. Last node is always tail (and head is always first node). But I know that such nodes are created internally by the implementation of the linked list. I leave this moment to the mentors.

Reply:
@ystankevych I didn't get your question :)

let's discuss it in the Slack chat